platforms/msx2/tools/MSXtk.h

```diff
@@ -0,0 +1,17 @@
+//_____________________________________________________________________________
+//   ▄▄   ▄ ▄  ▄▄▄ ▄▄ ▄   ▄▄▄▄           ▄▄         ▄▄ ▄ ▄  ▄▄
+//  ██ ▀ ██▀█ ▀█▄  ▀█▄▀    ██  ▄█▀▄ ▄█▀▄ ██   ██▀   ██▄▀ ▄  ██▀
+//  ▀█▄▀ ██ █ ▄▄█▀ ██ █    ██  ▀█▄▀ ▀█▄▀ ▀█▄ ▄██    ██ █ ██ ▀█▄
+//_____________________________________________________________________________
+//
+// by Guillaume "Aoineko" Blanchard (aoineko@free.fr)
+// available on GitHub (https://github.com/aoineko-fr/CMSXtk)
+// under CC-BY-AS license (https://creativecommons.org/licenses/by-sa/2.0/)
```

Comment:
There's a typo here and also in the other headers. I think we should open an issue on the upstream project to let @aoineko-fr know about it.

The typo is in the license name.
Where it says "CC-BY-AS", it should actually say "CC BY-SA".

Note: "SA" stands for "Share Alike"